plugins/DBSize/plugin.rb

```diff
@@ -0,0 +1,39 @@
+require 'sequel'
+require_relative '../../lib/BasePlugin/plugin.rb'
+require_relative '../../environment.rb'
+require_relative '../../models.rb'
+
+# Disk Sizes data plugin
+class DBSize < BasePlugin
+  def initialize
+    @@name = 'DBSize'
+    @@resource_name = 'node'
+    @@units = 'bytes'
+    @@table = :db_size_measurements
+    @@db_column = :db_size_ver
+    @@migrations_dir = File.dirname(__FILE__) + '/migrations'
+    @@database = Iam.settings.DB
+    @cache = Cache.new(ENV['CACHE_FILE'])
+    register
+  end
+
+  def store(fqdn)
+    # Pull node information from cache as a ruby hash
+    node_info = @cache.get(fqdn)
+    db_key = 'Data Base Size in Bytes'
+    # Check for valid data
+    if node_info[db_key].nil? || node_info[db_key] == ''
+      raise "No DBSize information for #{fqdn}\n"
+    end
+
+    # Insert data into disk_size_measurements table
+    @@database[@@table].insert(
+      node:          fqdn,
```

Comment:
I don't know if databases have FQDNs. The way it's currently setup each database is stored in the cache as such:
```
"database_name":"######"
```
where `#####` is the database size in bytes. A full FQDN would be something like `mysql://username:password@db_host/database_name`.
If you think that's a better way to store the database in the cache we can make that change, just a heads up.